Constants.java

```diff
@@ -0,0 +1,19 @@
+package com.company;
+
+public class Constants {
+
+        static final String FILE_TYPE_TXT = ".txt";
+        static final String FILE_TYPE_JSON = ".json";
+        static final String REMOVE_SPECIAL_CHAR = "[^a-zA-Z0-9 -]";
+        static final String SINGLE_SPACE = "";
+        static final String SUCCESS = "SUCCESS";
+        static final String FAILURE = "FAILURE";
+        static String errorMessage = null;
+        static String errorMessage1 = "Both file path and Search word must contain some value";
+        static String errorMessage2 = "File does not contain any data";
+        static String errorMessage3 = "File does not exist";
+        static String errorMessage4 = "File format not supported";
+        static String errorMessage5 = "Word not found";

```

Comment:
The error messages should have proper variable names. Like
ERROR_MESSAGE_WORD_NOT_FOUND etc

First of all, move the error messages to a separate ErrorMessages class, then you can refer to them as
ErrorMessages.WORD_NOT_FOUND
ErrorMessages.FILE_NOT_SUPPORTED 
etc

Reply:
Sure. Will do.